VAMobile/documentation/docs/Engineering/FrontEnd/DevelopmentBestPractices.md

```diff
@@ -0,0 +1,130 @@
+- `bugfix/1234-description-of-bug-being-fixed`
+- `hotfix/1234-description-of-critical-fix`
+
+After completing work in your branch, create a PR as described in "Pull Requests" below.
+
+### Sources
+
+- [Naming git branches](https://gist.github.com/jefffederman/1d492f98b8e3913a75ca)
+- [Branch naming](https://dev.to/varbsan/a-simplified-convention-for-naming-branches-and-commits-in-git-il4)
+
+## Pull Requests
+
+A Pull Request (PR) is a request to merge code. All pull requests must include certain information. Each PR must pass a series of manual and automated checks before it can be merged.
+
+### Pull request process
+
+Each pull request moves through a series of steps.
+
+1. Creation: an engineer creates a PR. See "Pull request creation" below.
+2. Review: a different engineer reviews the PR. See "Pull request review" below.
+3. Approval: when the reviewer is satisfied, they approve the PR and change the pipeline to "With QA (pre-develop)" so QA knows to begin testing.
+4. QA: a QA engineer tests the branch and updates TestRail. QA informs the PR creator of any issues they find. On approval, QA comments in the ticket and changes the pipeline to "Ready to merge to develop".
+5. Merge: the PR creator hits "Merge pull request" to merge their branch into the develop branch. On the ticket, the PR creator changes the pipeline to "With QA (develop)" and selects the current release under "Releases".
+6. Post-merge: QA completes final testing and closes the ticket.
+
+### Pull request creation
+
+When creating a PR, fill out each section in the PR template, including:
+
+- Description: provide context so a code reviewer or future developer understands why the PR exists and what it does
+- Screenshots: add before/after screenshots or videos showing the changes
+- Testing information: confirm testing on iOS and Android. Document any special considerations like edge cases or areas of focus for QA
+- Reviewer validations: tell reviewers what to check. You can usually copy the Acceptance Criteria from the ticket
+- Complete all PR checklist items
+
+:::info
+If you're actively working on your branch, create a draft PR. Move the PR out of draft status when it's complete and ready for review.
+:::
+
+Immediately after creating the PR, click the "Connect issue" button and choose the associated ticket. You must be logged into ZenHub to see the "Connect issue" button.
+
+:::important
+It's critical to link the PR to the ticket with the "Connect issue" button so we can track the work performed.
+:::
+
+If your PR is especially complex, you can annotate the PR with comments to help reviewers understand why you made particular decisions.
+
+When you create a PR, GitHub Actions automatically runs linting checks, unit tests, and E2E tests on your branch. Fix any failures, and make sure unit and E2E tests are modified or extended to cover your changes. See our [Unit Tests documentation](/docs/Engineering/FrontEnd/Testing/UnitTesting) for best practices.
```

Comment:
May be worth having a note to revert to Draft status if appropriate if tests are failing as it is not really ready for review.

Reply:
Added